test/e2e/apps/job.go

```diff
@@ -41,11 +41,11 @@ var _ = SIGDescribe("Job", func() {
 		By("Creating a job")
 		job := framework.NewTestJob("succeed", "all-succeed", v1.RestartPolicyNever, parallelism, completions, nil, backoffLimit)
 		job, err := framework.CreateJob(f.ClientSet, f.Namespace.Name, job)
-		Expect(err).NotTo(HaveOccurred())
+		Expect(err).NotTo(HaveOccurred(), "failed to create Job in namespace: %s", f.Namespace.Name)
```

Comment:
nit: Why Job is upercase? There are many this kind of words in the following also.

Reply:
I read [here](https://kubernetes.io/docs/contribute/style/style-guide/) that API objects should be capitalized. Was not sure whether it applies to code as well. Should I use small case?